neo/Network/test_node_leader.py

```diff
@@ -141,6 +139,7 @@ def mock_send_msg(node, message):
         with patch('twisted.internet.reactor.connectTCP', mock_connect_tcp):
             with patch('twisted.internet.reactor.callLater', mock_call_later):
                 with patch('neo.Network.NeoNode.NeoNode.SendSerializedMessage', mock_send_msg):
+                    leader.Shutdown()
```

Comment:
this was the only place that seemed to work. I tried placing it in the `teardown` and just as final statement for tests using the leader instance, but no luck. Don't know why, but this worked